### PR TITLE
Atomic/backends/_containers_storage.py: stop

### PR DESCRIPTION
**Pull request by baude**
Implement stop for a container.

## Related Bugzillas
-
-

## Related Issue Numbers
-
-

## Pull Request Checklist:

If your Pull request contains new features or functions, tests are required. If the PR is a bug fix and no tests exist, please consider adding some to prevent regressions.
- [ ] Unittests
- [ ] Integration Tests

### CODE REVIEW

**Comment by rhatdan** [comment]
LGTM

File: Atomic/backends/_containers_storage.py [hunk 1]
```diff
@@ -250,7 +250,10 @@ def start_container(self, name):
         raise UnderDevelopment()
 
     def stop_container(self, con_obj, **kwargs):
-        raise UnderDevelopment()
+        # If we plant to honor stop labels here, we will
```
Comment (TomSweeneyRedHat) [inline Atomic/backends/_containers_storage.py]: nit, "plant to" to  "plan to" or just drop both words and go to "If we honor stop..."

**Comment by baude** [comment]
bot, retest this please

**Comment by baude** [comment]
bot, retest this please

**Comment by baude** [comment]
bot, retest this please

**Comment by baude** [comment]
bot, retest this please

**Comment by baude** [comment]
@rh-atomic-bot 8483aa8

**Comment by baude** [comment]
@rh-atomic-bot r+ 8483aa8

**Pull request closed without merging**